src/main/java/jenkins/plugins/office365connector/FactsBuilder.java

```diff
@@ -79,46 +81,69 @@ public void addRemarks() {
         addFact(NAME_REMARKS, joinedCauses);
     }
 
-    public void addCommitters() {
+    public void addCommitters(boolean mentionCommitters) {
         if (!(run instanceof RunWithSCM)) {
             return;
         }
         RunWithSCM runWithSCM = (RunWithSCM) run;
         Set<User> authors = runWithSCM.getCulprits();
 
         String joinedCommitters = authors.stream()
-                .map(User::getFullName)
-                .collect(Collectors.joining(", "));
+            .map(user -> mentionCommitters ? TeamsMentionUtils.mentionUser(user) : user.getFullName())
```

Comment:
it complains here because you haven't extended examples in JSON file - same as for previous PR, please add it